### PR TITLE
Lineup saving and loading

### PR DESCRIPTION
**Pull request by DEAFPS**
- made `.prac` a toggle command (no need to .exitprac to exit)
- added `.god` to toggle godmode on and off

- `.savenade <name> <optional description>` //saves a lineup
- `.loadnade <name>` //loads a lineup
- `.deletenade <name>` //deletes a lineup from file
- `.importnade <code>` //uppon saving a lineup a code will be printed to chat, alternatively those can be retrieved from the savednades.cfg
- `.listnades <optional filter>` //lists either all saved lineups ever or if given a filter only those that match the filter

### CODE REVIEW

File: PracticeMode.cs [hunk 1]
```diff
@@ -114,6 +116,285 @@ private void HandleSpawnCommand(CCSPlayerController? player, string commandArg,
...
+			ReplyToUserCommand(player, "There are no saved nades!.");
+		}
+	}
+
+	
+	private void HandleLoadNadeCommand(CCSPlayerController? player, string loadNadeName)
+	{
+		if (!isPractice || player == null) return;
+		
+		if (!IsPlayerAdmin(player)) return;
```
Comment (shobhit-pathak) [inline PracticeMode.cs:314]: Hey, is there any reason why only admins can use these commands?
I think normal players should also be able to load nades and use other commands, right?

**Comment by shobhit-pathak** [comment]
Hey @DEAFPS,
Thanks again for your contribution! :D
I had a doubt and added a comment for the same, it would be great if you can help me out with that!

File: PracticeMode.cs (hunk 1 above)
Comment (DEAFPS) [reply PracticeMode.cs:314]: right, i suppose saving and loading can be unlocked for non-admin players but ill suggest to leave .deletenade admin only for now so players wont delete a important lineup by accident.

for a future PR i will figure out a way to have non-admin users have their own private lineups as well as the globally saved ones by admins

**Comment by DEAFPS** [comment]
i lifted IsPlayerAdmin of most of the commands apart from the `.deletenade` one. regulars will now be prompted to use a different name if a lineup with the given name already exists, admins will get a additional prompt that allows them to delete that lineup

**Review by shobhit-pathak — APPROVED** [review]

**Pull request merged**